### DISCUSSION TITLE
New Unenrollment Method

### DISCUSSION THREAD

**Discussion by schoolexploitkid** [3kh0/ext-remover#1409] [Exploits JOIN TITANIUM NETWORK TO SHARE PLEASE]
https://github.com/schoolexploitkid/De-Enterprise-unenrollment

**Comment by akabutnicer** [comment]
underrated method

**Comment by S-PScripts** [comment]
You must have money as well 🤑🤑

**Reply by schoolexploitkid** [reply]
a personal chromebook is worth prob more than like $12

**Reply by akabutnicer** [reply]
ive never heard such eloquent words before. this guy's the real deal

**Reply by schoolexploitkid** [reply]
yet i still cant get the samyk role bc they are like "u must send a video" and blah blah blah
